controlplane/src/core/repositories/SubgraphRepository.ts

```diff
@@ -248,6 +252,17 @@ export class SubgraphRepository {
           .execute();
       }
 
+      if (data.websocketSubprotocol !== undefined && data.websocketSubprotocol !== subgraph.websocketSubprotocol) {
```

Comment:
We could also set it to auto by default and always have a value instead of null AND auto for auto or one of the subprotocols.

Reply:
Done, doing it while creating the subgraph